CoherenceSpectrogramViewer/Source/CoherenceVisualizer.cpp

```diff
@@ -976,6 +978,8 @@ void CoherenceVisualizer::beginAnimation()
 	alphaE->setEditable(false);
 	CoherenceViewer->setEnabled(false);
 	SpectrogramViewer->setEnabled(false);
+    fstartEditable->setEditable(false);

```

Comment:
Changed so foi can only be changed while not getting data. Still not sure how I feel about this...